share/doc/help.rst

```diff
@@ -0,0 +1,15 @@
+
+Support
+-------
+If you find bugs, weird behaviour or just incomprehensible instructions,
+please open an issue
+`here <https://github.com/votca/xtp-tutorials/issues>`__
+
+We will try to fix it as fast as possible. For other questions, we provide an email list:
+
+`google
+group <https://groups.google.com/forum/?hl=de#!forum/votca-xtp>`__
+
```

Comment:
This forum is dead. We are using `votca` for this , too.

Plus I added it to the main page as well

Reply:
Thanks for let me know!